db/migrate/20170216193758_add_vm_table_comments.rb

```diff
@@ -0,0 +1,81 @@
+class AddVmTableComments < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines"
+
+    change_column_comment :vms, :autostart, "A boolean value that indicates whether or not the VM is set to autostart."
+    change_column_comment :vms, :availability_zone_id, "FK to the AVAILABILITY_ZONES table."
```

Comment:
Why all-caps on the table name?  The tables are lowercased and if you actually refer to them in queries this way it will fail, so this seems confusing